app/commands/tooling/handle_representer_deploy.rb

```diff
@@ -0,0 +1,12 @@
+class Tooling::HandleRepresenterDeploy
+  include Mandate
+
+  queue_as :default
+
+  initialize_with :track
+
+  def call
+    # TODO: update track's representer version
+    # TODO: trigger re-running of representer if representer version changes
```

Comment:
OK, I think you want this:

```ruby
Exercise::Representation.where(track:).with_feedback.includes(source_submission: :exercise).find_each do |representation|
  submission = representation.source_submission
  Submission::Representation::Init.(submission, type: :exercise, git_sha: submission.exercise.git_sha, run_in_background: true)
end
```

I suggest rebasing this on top of #3041 and then that **should** work. We're going to need to pair tomorrow to test all this though - it's really hard to work out whether anything is working as expected as there's so many bits that need setting (e.g. versions!)